README.md

```diff
@@ -376,10 +402,7 @@ The tests and benchmarks have their own documentation:
 - [Documentation of the tests](test/README.md)
 - [Documentation of the benchmarks](benchmark/README.md)
 
+## Contributions
 
-## Contributing to the project
-
-SYCL-BLAS is an Open Source project maintained by the HPCA group and
-Codeplay Software Ltd.
-Feel free to create an issue on the Github tracker to request features or
-report bugs.
+Please see the file CONTRIBUTING.md for further details if you would like to
```

Comment:
SYCL-BLAS doesn't currently have a CONTRIBUTING file.

Once made, it would be good if this was a link to the CONTRIBUTING file.

Reply:
I've added a CONTRIBUTING file